File: userservice/src/main/java/hu/bme/aut/classifiedadvertisementsite/userservice/controller/exceptions/ForbiddenException.java
package hu.bme.aut.classifiedadvertisementsite.userservice.controller.exceptions;

import org.springframework.http.HttpStatus;

public class ForbiddenException extends AbstractException {

    public ForbiddenException(String errorMessage) {
        super(errorMessage);
    }

    @Override
    public HttpStatus getHttpStatus() {
        return HttpStatus.FORBIDDEN;
    }
}

File: advertisementservice/src/main/kotlin/hu/bme/aut/classifiedadvertisementsite/advertisementservice/controller/exception/ForbiddenException.kt
package hu.bme.aut.classifiedadvertisementsite.advertisementservice.controller.exception

import org.springframework.http.HttpStatus

class ForbiddenException(errorMessage: String) : AbstractException(errorMessage) {
    override fun getHttpStatus(): HttpStatus {
        return HttpStatus.FORBIDDEN
    }
}